### ISSUE TITLE
CLion CMake error while opening nRF-Connect-SDK project

### ISSUE THREAD

**Issue by ilyaTanko** [zephyrproject-rtos/zephyr#42411]
Hello! 
I'm trying to reload CMake in an nRF-Connect-SDK project in the CLion IDE. 

TL;DR: the error is:
> CMake File API: D:\Programming\Embedded\Projects\SES\ncs\custom_project\custom_project\build_clion: no reply dir found

I've been using this guide:
https://blog.jetbrains.com/clion/2021/04/using-nrf52-with-cmake-connect-sdk/

My current toolchain and CMake settings are like these (the SDK version is 1.6.1):
![image](https://user-images.githubusercontent.com/22035673/152178808-8cf492e7-ee5c-41b6-890b-9d81c6314932.png)
![image](https://user-images.githubusercontent.com/22035673/152178824-963835c3-238c-4d04-8a2a-12c18d7da217.png)

The project starts reloading, it detects zephyr, west, etc., the build directory is created, but eventually it stumbles upon this error:

**CMake File API: D:\Programming\Embedded\Projects\SES\ncs\custom_project\custom_project\build_clion: no reply dir found**

I can see that in D:\Programming\Embedded\Projects\SES\ncs\custom_project\custom_project\build_clion\.cmake\api\v1 there is only "query" directory, but no "reply".
"query" directory contains these empty files:

![image](https://user-images.githubusercontent.com/22035673/152178857-b0e1cbca-74f6-4b13-a11a-67e71c6ae71a.png)

I'll list the CMake output from the stage where it fails here:

`-- libmetal version: 1.0.0 (D:/Programming/Embedded/Projects/SES/ncs/custom_project/custom_project)
-- Build type:  Debug
-- Host:    Windows/AMD64
-- Target:  Generic/arm
-- Machine: cortexm
-- Looking for include file stdatomic.h
-- Looking for include file stdatomic.h - found
-- open-amp version: 1.0.0 (D:/Programming/Embedded/Projects/SES/ncs/modules/lib/open-amp/open-amp)
-- Host:    Windows/AMD64
-- Target:  Generic/arm
-- Machine: cortexm
-- C_FLAGS :  -Wall -Wextra
-- Looking for include file fcntl.h
-- Looking for include file fcntl.h - found
CMake Warning at D:/Programming/Embedded/Projects/SES/ncs/zephyr/CMakeLists.txt:1635 (message):
  

        The CMake build type was set to 'Debug', but the optimization flag was set to '-Os'.
        This may be intentional and the warning can be turned off by setting the CMake variable 'NO_BUILD_TYPE_WARNING'

Partition 'mcuboot' is not included in the dynamic resolving since it is statically defined.
Partition 'mcuboot_pad' is not included in the dynamic resolving since it is statically defined.
Partition 'mcuboot_primary' is not included in the dynamic resolving since it is statically defined.
Partition 'mcuboot_primary_app' is not included in the dynamic resolving since it is statically defined.
Partition 'mcuboot_secondary' is not included in the dynamic resolving since it is statically defined.
Partition 'pcd_sram' is not included in the dynamic resolving since it is statically defined.
Partition 'rpmsg_nrf53_sram' is not included in the dynamic resolving since it is statically defined.
-- Configuring done
CMake Warning (dev) at D:/Programming/Embedded/Projects/SES/ncs/zephyr/cmake/linker/ld/target.cmake:33 (add_custom_command):
  Policy CMP0116 is not set: Ninja generators transform DEPFILEs from
  add_custom_command().  Run "cmake --help-policy CMP0116" for policy
  details.  Use the cmake_policy command to set the policy and suppress this
  warning.
Call Stack (most recent call first):
  D:/Programming/Embedded/Projects/SES/ncs/zephyr/CMakeLists.txt:1198 (configure_linker_script)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at D:/Programming/Embedded/Projects/SES/ncs/zephyr/cmake/linker/ld/target.cmake:33 (add_custom_command):
  Policy CMP0116 is not set: Ninja generators transform DEPFILEs from
  add_custom_command().  Run "cmake --help-policy CMP0116" for policy
  details.  Use the cmake_policy command to set the policy and suppress this
  warning.
Call Stack (most recent call first):
  D:/Programming/Embedded/Projects/SES/ncs/zephyr/CMakeLists.txt:1253 (configure_linker_script)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Generating done
-- Build files have been written to: D:/Programming/Embedded/Projects/SES/ncs/custom_project/custom_project/build_clion/build
CMake File API: D:\Programming\Embedded\Projects\SES\ncs\custom_project\custom_project\build_clion: no reply dir found

[Failed to reload]`

**Comment by carlescufi** [comment]
Sorry, but we don't support CLion officially, and this is the Zephyr project, not Nordic's nRF Connect SDK. You can either post this in a GitHub discussion referring instead to a version of Zephyr or post the question in the Nordic Devzone.
